src/Nethermind/Directory.Build.props

```diff
@@ -1,7 +1,9 @@
 <Project>
 
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Configuration Condition="'$(Configuration)' == ''">Release</Configuration>
```

Comment:
I was able to reproduce the issue but changing it from Debug to Release didn't help. Building from command line works just fine though. What the heck is Visual Studio doing?